### PR TITLE
Added Student data

### PR DESCRIPTION
**Pull request by arihantbansal**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](url) https://arihantbansal.github.io/fossasiagci18/

## Description / Changes

Added name and other required details of student which are to be displayed on the FOSSASIA GCI '18 website, as part of a GCI task.

### Before:
old

### After:
new

- - - - - - - - - - - -

### CODE REVIEW

**Comment by sk9331657** [comment]
@arihantbansal  Update the PR.

**Comment by arihantbansal** [comment]
Added details.

**Comment by arihantbansal** [comment]
Please check now.

File: _data/students.yml [hunk 1]
```diff
@@ -105,3 +105,9 @@
 - name: Akshit Goyal
   github: akshit15013
   image: AkshitGoyal.jpg
+
+- name: Arihant Bansal
+  github: arihantbansal
+  image: rsz_arihant.png
+  lat: 19.130970
+  lng: 72.817681
```
Comment (sakshee-19) [inline _data/students.yml]: Add EOF

**Comment by sk9331657** [comment]
@simsausaurabh  The problem is with outdated codebase pr contains. Squashing can be done on our end.

**Comment by simsausaurabh** [comment]
@sk9331657 But it isn't a good practice to squash the commits on this end.

**Comment by arihantbansal** [comment]
Pulled the latest codebase and added EOF

**Comment by arihantbansal** [comment]
I cannot squash the commit as I don't have write access to the repository.

**Comment by sakshee-19** [comment]
@arihantbansal You need to squash the commits locally and then push a single commit. You don't need to have right access to repo for that. They are totally independent things.

**Comment by sakshee-19** [comment]
@arihantbansal Try ```https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git``` and if you face any issue, please ask.

**Comment by arihantbansal** [comment]
All conflicts resolved.

**Comment by arihantbansal** [comment]
I'm unable to squash commits since I was on the go and made my commits on the Github website itself.

**Comment by fragm3** [comment]
@arihantbansal make a new PR with updated codebase, it would be faster that way.

**Pull request closed without merging**